schemas/context/profile.schema.json

```diff
@@ -94,20 +94,23 @@
         "xdm:preferredLanguage": {
           "title": "Preferred Language",
           "type": "string",
```

Comment:
Do we allow keeping a language preference order like any browser or operating system? My preference order would be: `["de_DE", "de", "en_US", "en"]`, i.e. "real" German first, then any form of German, then US English, then any form of English?